README.md

```diff
@@ -55,13 +55,37 @@ Install
         BaoPham\DynamoDb\DynamoDbServiceProvider::class,
         ...
     ];
-    ```
-
-* Run:
+    ``` 
+    
+* Laravel
 
     ```php
+    php artisan vendor:publish
+    ```    
+    
+* Lumen
+  
+  - Set up a `config_path` helper: https://gist.github.com/mabasic/21d13eab12462e596120
```

Comment:
Or you can fix `DynamoDbServiceProvider` replacing `config_path()` to `app()->path()`. It works in Lumen and Laravel :+1:

Reply:
Thanks @gchauqui-bondacom ! `app()->path()` makes it works. It should be cool if the pakage change this so then itsn't necessary to register `config_path` helper. What do you think @baopham  ?

Reply:
yep, PR is welcome.